src/actions/roles.test.js

```diff
@@ -44,43 +44,6 @@ describe('Actions.Roles', () => {
         assert.deepEqual(roles.system_admin.permissions, TestHelper.basicRoles.system_admin.permissions);
     });
 
-    it('getRoleByName', async () => {
-        nock(Client4.getRolesRoute()).
```

Comment:
Can you add these back along with the actions and modify them to accordingly for removal of status flags?